test/python/quantum_info/operators/test_utils.py

```diff
@@ -0,0 +1,49 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests utility functions for operator classes."""
+
+import unittest
+
+from qiskit.quantum_info import SparsePauliOp, anti_commutator, commutator, double_commutator
+from qiskit.test import QiskitTestCase
+
+
+class TestOperatorUtils(QiskitTestCase):
```

Comment:
It might be worthwhile adding one additional test in each case to ensure that the method does not _always_ return zero-ops. Because if this were to break in the future, these tests would not catch that.